pkg/resourcemanager/webhook/podschedulername/handler.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Did you think about adding a simple envtest for this new webhook? The implementation obviously is quite trivial and straight-forward and you already have some great unit tests for it, but maybe it's worth having such small integration test?

Reply:
Thanks for the reminder! I added envtest with the new commit.